src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -49,7 +49,7 @@ namespace Microsoft.ML.Transforms.Text
     /// <summary>
     /// <see cref="ITransformer"/> resulting from fitting a <see cref="LatentDirichletAllocationEstimator"/>.
     /// </summary>
-    public sealed class LatentDirichletAllocationTransformer : OneToOneTransformerBase
+    public sealed class LatentDirichletAllocationTransformer : OneToOneTransformerBase, IDisposable
```

Comment:
⚠️ This is the "ugly" part of this change for me. If someone is upcasting this to `ITransformer`, how will they know to dispose of it?

Reply:
I'm leaving this change for a separate pull request.